From 5c0a8cb8636134d6e0836d0706ee9467b468ee95 Mon Sep 17 00:00:00 2001
From: "Fabio M. De Francesco" <fmdefrancesco@gmail.com>
Date: Fri, 8 Jul 2022 22:24:54 +0200
Subject: [PATCH] highmem: Delete inconsistent sentences from kdocs

Both kdocs of kmap_local_page() and kmap_local_folio() contain one same
sentence which is inconsistent with the fact that the local mappings
should be preferred over kmap() and kmap_atomic().

Therefore, delete "Only use when really necessary." from kdocs of
kmap_local_page and kmap_local_folio() in highmem.h.

Cc: Thomas Gleixner <tglx@linutronix.de>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Suggested-by: Ira Weiny <ira.weiny@intel.com>
Signed-off-by: Fabio M. De Francesco <fmdefrancesco@gmail.com>
---
 include/linux/highmem.h | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/include/linux/highmem.h b/include/linux/highmem.h
index fee9835e3793..dffd37966f07 100644
--- a/include/linux/highmem.h
+++ b/include/linux/highmem.h
@@ -86,8 +86,7 @@ static inline void kmap_flush_unused(void);
  * temporarily mapped.
  *
  * While it is significantly faster than kmap() for the higmem case it
- * comes with restrictions about the pointer validity. Only use when really
- * necessary.
+ * comes with restrictions about the pointer validity.
  *
  * On HIGHMEM enabled systems mapping a highmem page has the side effect of
  * disabling migration in order to keep the virtual address stable across
@@ -120,8 +119,7 @@ static inline void *kmap_local_page(struct page *page);
  * temporarily mapped.
  *
  * While it is significantly faster than kmap() for the higmem case it
- * comes with restrictions about the pointer validity. Only use when really
- * necessary.
+ * comes with restrictions about the pointer validity.
  *
  * On HIGHMEM enabled systems mapping a highmem page has the side effect of
  * disabling migration in order to keep the virtual address stable across
-- 
2.36.1

